CWE/common/countries/Chechnya.txt

```diff
@@ -12,18 +23,18 @@ party = {
 		end_date = 2100.1.1
 }
 party = {
-		name = "party_conservative_0_CHC"
+		name = "UPF "Islamic Order""
```

Comment:
Party names cannot have commas within commas

Reply:
Fixed it.